Sources/KlaviyoSwift/Models/ProfileAPIExtension.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  File.swift
+//
+//
+//  Created by Ajay Subramanya on 8/6/24.
+//
+
+import Foundation
+import KlaviyoCore
+
+extension String {
+    fileprivate func returnNilIfEmpty() -> String? {
+        isEmpty ? nil : self
+    }
+}
+
+extension Profile {
```

Comment:
This is the extension mentioned in the PR description that helps map the public facing API to API models.